Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +11,30 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            if (message == null)
+                return message;
+
+            var byteArray = ASCIIEncoding.ASCII.GetBytes(message.ToCharArray());
+            for (int i = 0; i < message.Length; i++)
+            {
+                byteArray[i] += shift;
+            }
+
+            return new string(Encoding.ASCII.GetChars(byteArray));
         }
 
         public static string Decrypt(string message, byte shift)
```

Comment:
Please redo the decrypt method: what's the difference between encrypt and decrypt 😄 ?